src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -64,9 +64,12 @@ public void after() {
     TestUtils.dropTable(topicName);
   }
 
-  @Test
-  public void testSinkTask() throws Exception {
+  /** @param useOAuth true if using oauth as authentication, otherwise use snowflake_jwt */
+  private void sinkTaskTest(boolean useOAuth) throws Exception {
```

Comment:
Good idea to test with and without, but try using this [Parameterized framework](https://github.com/snowflakedb/snowflake-kafka-connector/blob/master/src/test/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientProviderTest.java#L54) instead. I believe you can set it to just 'false' for now, and then add true later

Reply:
Thanks for the reference! Applied parameterized framework.